docker/docker-compose.yml

```diff
@@ -8,18 +8,23 @@
 
 version: "3.7"
 
+networks:
+  docker_mainflux-base-net:
+    external: true
+
 services:
   mfxkit:
+    image: mainflux/mfxkit:latest
```

Comment:
Image still calls `mfkit` ? Not really sure if we need compose placeholder  (which can't work without the rest of Mainflux) ?

Reply:
What do you mean? This works with the rest of Mainflux

Reply:
Yea, I was thinking do we need a docker-compose in the service template? 
We use a single docker-compose on the Mainflux level, not per service. So this composition can't work independently right (e.g this new service that you make from this template, needs to be added in `maifnlix/docker/docker-compose`)? If so, this file is not usable.

Reply:
I think so. It's useful to build and run the system fast without having to enter in "native" issues. We do use docker-compose per service for addons. This is a sort of external addon.

Reply:
Ok. Resolved.